nifi-framework-api/src/main/java/org/apache/nifi/controller/status/history/StatusHistory.java

```diff
@@ -41,4 +41,9 @@
      * @return List of snapshots for a given component
      */
     List<StatusSnapshot> getStatusSnapshots();
+
+    /**
+     * @return <code>true</code> if counter values are included in the Status History
+     */
+    boolean isIncludeCounters();
```

Comment:
If we're able to remove the flag from the `StatusHistoryDTO`, we may also be able to remove this one.